src/decorator/options/CalculatedColumnOptions.ts

```diff
@@ -0,0 +1,31 @@
+import { ColumnType } from "../../driver/types/ColumnTypes"
+import { ValueTransformer } from "./ValueTransformer"
+
+/**
+ * Describes all calculated column's options.
+ */
+export interface CalculatedColumnOptions {
+    /**
+     * Column type. Must be one of the value from the ColumnTypes class.
+     */
+    type?: ColumnType
+
+    /**
+     * Return type of HSTORE column.
+     * Returns value as string or as object.
+     */
+    hstoreType?: "object" | "string"
+
+    /**
+     * Query to be used to populate the column data. This query is used when generating the relational db script.
+     * The query function is called with the current entities alias either defined by the Entity Decorator or automatically
+     * @See https://github.com/typeorm/typeorm/issues/9323 for more details.
```

Comment:
Should be rather a link to docs.

Reply:
I saw another issue did it this way so I replicated theirs but I agree that a link to the docs would be better for this. I will update this as well today. Thanks for all your feedback

Reply:
This has been updated @Ginden